src/Storages/RocksDB/EmbeddedRocksDBBulkSink.cpp

```diff
@@ -167,16 +168,15 @@ std::pair<ColumnString::Ptr, ColumnString::Ptr> EmbeddedRocksDBBulkSink::seriali
         auto & serialized_value_offsets = serialized_value_column->getOffsets();
         WriteBufferFromVector<ColumnString::Chars> writer_key(serialized_key_data);
         WriteBufferFromVector<ColumnString::Chars> writer_value(serialized_value_data);
-
-        for (auto && chunk : input_chunks)
+        FormatSettings format_settings; /// Format settings is 1.5KB, so it's not wise to create it for each row
+        for (const auto & chunk : input_chunks)
```

Comment:
why was this changed?

Reply:
Should be a mistake when I cherry-pick the commits